effortless-chef-io/content/effortless/variables_and_config.md

```diff
@@ -58,6 +58,9 @@ ssl_verify_mode
 verify_api_cert
 : Boolean option to determine if the API certification should be verified. Default is `false`
 
+scaffold_rubygems_url
```

Comment:
I think this should just be `rubygems_url` as it's in a hab config not a plan variable.

Reply:
Latest commit should fix this